v1/website/pages/en/users.js

```diff
@@ -9,26 +9,27 @@ const React = require('react');
 const CompLibrary = require('../../core/CompLibrary.js');
 
 const Container = CompLibrary.Container;
+const Showcase = CompLibrary.Showcase;
 const siteConfig = require(`${process.cwd()}/siteConfig.js`);
 const translate = require('../../server/translate.js').translate;
 
 class Users extends React.Component {
-  renderUser(user) {
-    return (
-      <a href={user.infoLink} key={user.infoLink}>
-        <img src={user.image} alt={user.caption} title={user.caption} />
-      </a>
-    );
+  getUsersToShowcase() {
```

Comment:
The original way, although it needs 2 passes of the array, is much more declarative. Please revert to that approach.